### ISSUE TITLE
Training does not start [if config path is incorrect] 

### ISSUE THREAD

**Issue by Denis-Pinkle** [DeepLabCut/DeepLabCut#999]
**Your Operating system and DeepLabCut version**
Please state your operating system, env, and which version of DeepLabCut you are using.

OS: Win 10
DeepLabCut Version:2.2b8
Anaconda env used: DLC-CPU

**Describe the problem**
I've created a project with DLC and labeled about 80 frames from 4 different videos (multi-animal). I've checked the labels, which looked good, and started a training dataset. But every time I want to train the network it starts until it says "starting multi-animal training ...." and i see that the CPU is doing some work for a couple of seconds but than it stops and kicks me out of the ipython session so that I'm back in my environment (DLC-CPU). It also creates the trainset95shuffle1 subfolder in the evaluation-results folder but it is empty. No Error message or something similar appears, it just stops. I've checked Hardware requirements and the problem should not be that i don't fulfill them. I'm running the code in the Command Prompt from Windows.

**Code output** 
C:\coding>activate DLC-CPU

(DLC-CPU) C:\coding>ipython
Python 3.7.9 (default, Aug 31 2020, 17:10:11) [MSC v.1916 64 bit (AMD64)]
Type 'copyright', 'credits' or 'license' for more information
IPython 7.19.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import deeplabcut

In [2]: config_path = '/coding/DLC/test/training_dataset/test1-Denis-2020-11-14/config.yaml'

In [3]: deeplabcut.train_network(config_path)
Selecting multi-animal trainer
Config:
{'all_joints': [[0], [1], [2], [3], [4], [5], [6], [7]],
 'all_joints_names': ['tentacletip1',
                      'tentacletip2',
                      'tentacletip3',
                      'tentaclebase1',
                      'tentaclebase2',
                      'tentaclebase3',
                      'hypostome',
                      'foot'],
 'batch_size': 8,
 'bottomheight': 400,
 'crop': True,
 'crop_pad': 0,
 'cropratio': 0.4,
 'dataset': 'training-datasets\\iteration-0\\UnaugmentedDataSet_test1Nov14\\test1_Denis95shuffle1.pickle',
 'dataset_type': 'multi-animal-imgaug',
 'deterministic': False,
 'display_iters': 500,
 'fg_fraction': 0.25,
 'global_scale': 0.4,
 'init_weights': 'C:\\coding\\envs\\DLC-CPU\\lib\\site-packages\\deeplabcut\\pose_estimation_tensorflow\\models\\pretrained\\resnet_v1_50.ckpt',
 'intermediate_supervision': False,
 'intermediate_supervision_layer': 12,
 'leftwidth': 400,
 'location_refinement': True,
 'locref_huber_loss': True,
 'locref_loss_weight': 0.05,
 'locref_stdev': 7.2801,
 'log_dir': 'log',
 'max_input_size': 1500,
 'mean_pixel': [123.68, 116.779, 103.939],
 'metadataset': 'training-datasets\\iteration-0\\UnaugmentedDataSet_test1Nov14\\Documentation_data-test1_95shuffle1.pickle',
 'min_input_size': 64,
 'minsize': 100,
 'mirror': False,
 'multi_step': [[0.0001, 7500], [5e-05, 12000], [1e-05, 200000]],
 'net_type': 'resnet_50',
 'num_joints': 8,
 'num_limbs': 0,
 'optimizer': 'adam',
 'pafwidth': 20,
 'pairwise_huber_loss': False,
 'pairwise_loss_weight': 0.1,
 'pairwise_predict': False,
 'partaffinityfield_graph': [],
 'partaffinityfield_predict': True,
 'pos_dist_thresh': 17,
 'project_path': 'C:\\coding\\DLC\\test\\training_dataset\\test1-Denis-2020-11-14',
 'regularize': False,
 'rightwidth': 400,
 'save_iters': 10000,
 'scale_jitter_lo': 0.5,
 'scale_jitter_up': 1.25,
 'scoremap_dir': 'test',
 'shuffle': True,
 'snapshot_prefix': '\\coding\\DLC\\test\\training_dataset\\test1-Denis-2020-11-14\\dlc-models\\iteration-0\\test1Nov14-trainset95shuffle1\\train\\snapshot',
 'stride': 8.0,
 'topheight': 400,
 'weigh_negatives': False,
 'weigh_only_present_joints': False,
 'weigh_part_predictions': False,
 'weight_decay': 0.0001}
Activating limb prediction...
Starting with multi-animal imaug + adam pose-dataset loader.
Batch Size is 8
Getting specs multi-animal-imgaug 0 8
Initializing ResNet
Loading ImageNet-pretrained resnet_50
2020-11-19 11:14:31.957286: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
Training parameters:
{'stride': 8.0, 'weigh_part_predictions': False, 'weigh_negatives': False, 'fg_fraction': 0.25, 'mean_pixel': [123.68, 116.779, 103.939], 'shuffle': True, 'snapshot_prefix': '\\coding\\DLC\\test\\training_dataset\\test1-Denis-2020-11-14\\dlc-models\\iteration-0\\test1Nov14-trainset95shuffle1\\train\\snapshot', 'log_dir': 'log', 'global_scale': 0.4, 'location_refinement': True, 'locref_stdev': 7.2801, 'locref_loss_weight': 0.05, 'locref_huber_loss': True, 'optimizer': 'adam', 'intermediate_supervision': False, 'intermediate_supervision_layer': 12, 'regularize': False, 'weight_decay': 0.0001, 'crop_pad': 0, 'scoremap_dir': 'test', 'batch_size': 8, 'dataset_type': 'multi-animal-imgaug', 'deterministic': False, 'mirror': False, 'pairwise_huber_loss': False, 'weigh_only_present_joints': False, 'partaffinityfield_predict': True, 'pairwise_predict': True, 'all_joints': [[0], [1], [2], [3], [4], [5], [6], [7]], 'all_joints_names': ['tentacletip1', 'tentacletip2', 'tentacletip3', 'tentaclebase1', 'tentaclebase2', 'tentaclebase3', 'hypostome', 'foot'], 'bottomheight': 400, 'crop': True, 'cropratio': 0.4, 'dataset': 'training-datasets\\iteration-0\\UnaugmentedDataSet_test1Nov14\\test1_Denis95shuffle1.pickle', 'display_iters': 500, 'init_weights': 'C:\\coding\\envs\\DLC-CPU\\lib\\site-packages\\deeplabcut\\pose_estimation_tensorflow\\models\\pretrained\\resnet_v1_50.ckpt', 'leftwidth': 400, 'max_input_size': 1500, 'metadataset': 'training-datasets\\iteration-0\\UnaugmentedDataSet_test1Nov14\\Documentation_data-test1_95shuffle1.pickle', 'min_input_size': 64, 'minsize': 100, 'multi_step': [[0.0001, 7500], [5e-05, 12000], [1e-05, 200000]], 'net_type': 'resnet_50', 'num_joints': 8, 'num_limbs': 0, 'pafwidth': 20, 'pairwise_loss_weight': 0.1, 'partaffinityfield_graph': [], 'pos_dist_thresh': 17, 'project_path': 'C:\\coding\\DLC\\test\\training_dataset\\test1-Denis-2020-11-14', 'rightwidth': 400, 'save_iters': 10000, 'scale_jitter_lo': 0.5, 'scale_jitter_up': 1.25, 'topheight': 400}
Starting multi-animal training....

(DLC-CPU) C:\coding>

Thank you very much for your help and for the very exciting toolkit that i hopefully can soon use for my work!
Cheers Denis

**Comment by nunorito** [comment]
Hey! 

I had a similar issue and what solved for me was updating my GPU drivers to the latest version. Also make sure you have the correct CUDA and tensorflow versions.

**Comment by MMathisLab** [comment]
the only thing I can see is `2020-11-19 11:14:31.957286: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2`; but you might consider just moving this training to the cloud; it will take ~5 days to train on your CPU, vs. ~5 hours here: https://github.com/DeepLabCut/DeepLabCut/blob/master/examples/COLAB_maDLC_TrainNetwork_VideoAnalysis.ipynb 

simply move your project folder to google drive, then follow the notebook to link your data to this. To note, create a new training dataset set on colb (i.e. you can make shuffle 2) -- otherwise at that step paths ar hard coded and it would be in windows, vs. the require linux style for colab. 

oh, edit to add: this is wrong for windows:

`config_path = '/coding/DLC/test/training_dataset/test1-Denis-2020-11-14/config.yaml'`

for windows: 

`config_path = r'C:\coding\DLC\test\training_dataset\test1-Denis-2020-11-14\config.yaml'`

please see here for formating: https://github.com/DeepLabCut/DeepLabCut/blob/master/docs/UseOverviewGuide.md#configure-the-project
